charts/group-sync-config/templates/configmap-ca.yaml

```
@@ -0,0 +1,17 @@
+{{- if .Values.groupSync }}
+{{- range .Values.groupSync }}
+{{- $groupSyncName := .name }}
+{{- range .providers }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $groupSyncName }}-{{ .name }}
+  labels:
+  {{- include "group-sync-config.labels" $ | nindent 4 }}  
+data:
+  ca.crt: |
+  {{- (include "apc-global-overrides.caCertificatesBundle" $) | nindent 4 }}
+---
+{{- end }}
+{{- end }}
+{{- end }}
```

Do we really need that many configmaps when only one caCert can be specified? It should be enough to have just one and reference that ..
```suggestion
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ include "group-sync-config.fullname" . }}-ca-bundle
  labels:
  {{- include "group-sync-config.labels" $ | nindent 4 }}  
data:
  ca.crt: |
  {{- (include "apc-global-overrides.caCertificatesBundle" $) | nindent 4 }}
```